modules/afsql/afsql.c

```diff
@@ -859,10 +845,19 @@ afsql_dd_should_begin_new_transaction(const AFSqlDestDriver *self)
   return afsql_dd_is_transaction_handling_enabled(self) && self->super.worker.instance.batch_size == 1;
 }
 
+static gint
+_batch_size(const AFSqlDestDriver *self)
+{
+  if (self->super.batch_lines <= 0)
+    return DEFAULT_SQL_TX_SIZE;
+
+  return self->super.batch_lines;
+}
```

Comment:
why don't we initialize batch_lines to DEFAULT_SQL_TX_SIZE? Also, I think by default the sql driver wasn't batching as it was also depending on the "explicit-transactions" flag (the name might be somewhat different).

Reply:
Well... it could be set, I just used a private function.
I did not change the default: whenever this function is called, the explicit-transaction flag is checked before.

Reply:
@bazsi I have found the flag: "explicit-commits". And you're right, it's another requirement for batching.
The other comment about default value placement seems to be optional for me, so I mark this as resolved.